servers/dsub/jobs/controllers/aggregation_controller.py

```diff
@@ -33,6 +38,12 @@ def get_job_aggregations(timeFrame, projectId=None):
     label_summaries = {}
 
     for job in jobs:
+        # AGGREGATION_FILTER_LABEL is a global config parameter only set for aggregation testing
+        # and jobs that do not have this label is discarded when testing
+        if 'AGGREGATION_FILTER_LABEL' in current_app.config and job.name.find(
+                current_app.config['AGGREGATION_FILTER_LABEL']) < 0:
```

Comment:
I was expecting the filter to be on a label key+value, but I guess name works too if that's easier.

If you're using the name though, you should rename this config value to something like "AGGREGATION_JOB_NAME_FILTER"